modules/machine-config-drift-detection.adoc

```diff
@@ -0,0 +1,106 @@
+
+If the MCD detects configuration drift, the MCD:
+
+* Emits an error to the console logs
+* Emits a Kubernetes event
+* Stops further detection on the node
+* Sets the node and MCP to `Degraded`
+
+You can check if you have a degraded node by listing the MCPs:
+
+[source,terminal]
+----
+$ oc get mcp worker
+----
+
+If you have a degraded MCP, the `DEGRADEDMACHINECOUNT` field is non-zero, similar to the following:
+
+.Sample output
+[source,terminal]
+----
+NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+worker   rendered-worker-404caf3180818d8ac1f50c32f14b57c3   False     True       True       2              1                   1                     1                      5h51m
+----
+
+You can determine if the problem is caused by configuration drift by examining the machine config pool:
+
+[source,terminal]
+----
+$ oc describe mcp worker
+----
+
+.Example output
+[source,terminal]
+----
+ ...
+    Last Transition Time:  2021-12-20T18:54:00Z
+    Message:               Node ci-ln-j4h8nkb-72292-pxqxz-worker-a-fjks4 is reporting: "content mismatch for file \"/etc/mco-test-file\"" <1>
+    Reason:                1 nodes are reporting degraded status on sync
+    Status:                True
+    Type:                  NodeDegraded <2>
+ ...
+----
+<1> The node that contains the configuration drift. And, the error message indicating that the contents of the `/etc/mco-test-file`, which was added by the machine config, has changed outside of the machine config.
+<2> The state of the node is `Degraded`.
+
+Or, if you know which node is degraded, examine that node:
+
+[source,terminal]
+----
+$ oc describe node/ci-ln-j4h8nkb-72292-pxqxz-worker-a-fjks4
+----
+
+.Example output
+[source,terminal]
+----
+ ...
+
+Annotations:        cloud.network.openshift.io/egress-ipconfig: [{"interface":"nic0","ifaddr":{"ipv4":"10.0.128.0/17"},"capacity":{"ip":10}}]
+                    csi.volume.kubernetes.io/nodeid:
+                      {"pd.csi.storage.gke.io":"projects/openshift-gce-devel-ci/zones/us-central1-a/instances/ci-ln-j4h8nkb-72292-pxqxz-worker-a-fjks4"}
+                    machine.openshift.io/machine: openshift-machine-api/ci-ln-j4h8nkb-72292-pxqxz-worker-a-fjks4 <1>
+                    machineconfiguration.openshift.io/controlPlaneTopology: HighlyAvailable
+                    machineconfiguration.openshift.io/currentConfig: rendered-worker-67bd55d0b02b0f659aef33680693a9f9
+                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-67bd55d0b02b0f659aef33680693a9f9
+                    machineconfiguration.openshift.io/reason: content mismatch for file "/etc/mco-test-file" <2>
+                    machineconfiguration.openshift.io/state: Degraded <3>
+ ...
+----
+<1> The node that contains the configuration drift.
```

Comment:
Is this callout necessary, because you already knew which node it was, per the oc command above.